solution/ui/regulations/eregs-vite/src/views/Resources.vue

```diff
@@ -311,71 +363,65 @@ export default {
             return returnArr;
         },
         getPartDict(dataQueryParams) {
-
             const parts = dataQueryParams.part.split(",");
-            const newPartDict = {}
+            const newPartDict = {};
 
-            parts.forEach(x => {
-                newPartDict[x] = {
+            parts.forEach((part) => {
+                newPartDict[part] = {
                     title: "42",
                     sections: [],
                     subparts: [],
                 };
-            })
+            });
 
-            this.partDict = newPartDict
+            this.partDict = newPartDict;
 
             if (dataQueryParams.section) {
-
                 const sections = dataQueryParams.section
                     .split(",")
-                    .filter(x =>
-                        x.match(/^\d+/) && x.match(/\d+$/)
-                    )
+                    .filter((x) => x.match(/^\d+/) && x.match(/\d+$/))
```

Comment:
echoing Phil's request for a descriptive variable name.   What is this supposed to be here?